docs/guides/admin/docs/modules/liveschedule.md

```diff
@@ -0,0 +1,169 @@
+
+Pre-requisites
+--------------
+
+To use this service, you need to have:
+
+1. A streaming server (Wowza, Adobe Media Server) or CDN already set up to stream live content
+2. A capture agent capable of streaming to it
+3. A player capable of playing live streams. The Paella player using the Flash component supports the rtmp protocol. Other players/protocols have not been tested.
+
+Configuration
+-------------
+
+### Step 1: Configure the service
+
+Edit  _etc/org.opencastproject.liveschedule.impl.LiveScheduleServiceImpl.cfg_.
+
+If you capture agent does not register a _capture.device.live.resolution.WIDTHxHEIGHT_ property, it's mandatory to configure the _live.streamingUrl_. 
+
+The _live.streamingUrl_ should be set to your streaming server url (or the subscriber url specified by your CDN).
+
+This is the url that the player will use to play the live stream.
+For instance, if using rtmp, set it to something like: rtmp://STREAMING_SERVER_HOST:PORT/STREAMING_APPLICATION/
+
+```
+# Configuration for the Live Schedule Service
+
+#
+# If the capture agent doesn't register the capture.device.live.resolution.WIDTHxHEIGHT property,
+# specify live.streamingUrl, live.resolution, and live.streamName below:
+#
+# -----------------------------
+
+# The streaming base url e.g. rtmp://streaming.server/live/
+#live.streamingUrl=rtmp://streaming.server/live
+
+# If a comma-separated list is informed, several resolutions will be generated for each flavor
+live.resolution=1920x540,960x270
+
+# Possible variable substitutions:
+# #{id} = media package id
+# #{flavor} = type-subtype of flavor
+# #{caName} = capture agent name
+# #{resolution} = video resolution e.g. 1920x1080
+#live.streamName=#{id}-#{flavor}.stream
+live.streamName=#{caName}-#{flavor}.stream-#{resolution}
+
+# -----------------------------
+
+# The same mime-type applies to all flavors and resolutions
+live.mimeType=video/x-flv
+
+# If a comma-separated list is informed, several streams links will be generated, one for each
+# resolution-targetFlavor combination.
+# Default is presenter/delivery
+#live.targetFlavors=presenter/delivery
+
+# The distribution service to use: download or aws.s3
+live.distributionService=download
+```
+
+### Step 2: Configure the capture agent
+
+#### Capture agent does not register the _capture.device.live.resolution.WIDTHxHEIGHT_ property 
+
+Configure the capture agent to stream to your streaming server (or the publisher url specified by your CDN), using the same stream name specified in live.streamName.
+
+#### Capture agent registers the _capture.device.live.resolution.WIDTHxHEIGHT_ property 
+
+If you develop the code that runs on your capture agent, instead of configuring the live.streamingUrl,
+live.resolution, live.streamName, 
+you can update the capture agent firmware to pass the following when registering to Opencast:
+
+- capture.device.names: add 'live' to the current list of devices
```

Comment:
You are correct. The LiveScheduler code does not use it. We included it for consistency, because all the capture agent properties have the 'device name' in them e.g. capture.device.names=presenter and capture.device.presenter.flavor=presenter/source. Honestly, I don't know if this is necessary or not.